src/main/java/reactor/kafka/receiver/internals/ConsumerFlux.java

```diff
@@ -113,26 +102,67 @@
 
         pollEvent = new PollEvent();
         commitEvent = new CommitEvent();
-        scheduler = Schedulers.single(receiverOptions.schedulerSupplier().get());
+        eventScheduler = KafkaSchedulers.newEvent(receiverOptions.groupId());
```

Comment:
Note for the reviewers:

the diff is misleading. `scheduler` got moved to `DefaultKafkaReceiver`, and `eventScheduler` was moved to a field from here:
https://github.com/reactor/reactor-kafka/pull/139/files#diff-56dd574c6a295ffb097b9cfebce0390cL179